src/atomate2/vasp/flows/lobster.py

```diff
@@ -0,0 +1,187 @@
+"""Flows for Lobster computations."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker
+from pymatgen.core import Structure
+
+from atomate2.lobster.jobs import PureLobsterMaker
+from atomate2.vasp.flows.core import DoubleRelaxMaker
+from atomate2.vasp.jobs.base import BaseVaspMaker
+from atomate2.vasp.jobs.core import RelaxMaker, StaticMaker
+from atomate2.vasp.jobs.lobster import (
+    VaspLobsterMaker,
+    delete_lobster_wavecar,
+    get_basis_infos,
+    get_lobster_jobs,
+    update_user_incar_settings_maker,
+)
+from atomate2.vasp.sets.core import StaticSetGenerator
+
+__all__ = ["LobsterMaker"]
+
+
+@dataclass
+class LobsterMaker(Maker):
+    """
+    Maker to perform a Lobster computation.
+
+    Optional optimization.
+    Optional static computation with symmetry
+    to preconverge the wavefunction.
+    Static computation with ISYM=0 is performed
+    Several Lobster computations testing several basis sets are performed.
+    The basis sets can only be changed with yaml files.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    bulk_relax_maker : .BaseVaspMaker or None
+        A maker to perform a relaxation on the bulk.
+        Set to ``None`` to skip the
+        bulk relaxation
+    additional_static_run_maker: .BaseVaspMaker or None
+        A maker to perform a preconvergence run
+        before the wavefunction computation without symmetry
+    vasp_lobster_maker : .BaseVaspMaker
+        A maker to perform the computation of the wavefunction before the static run.
+        Cannot be skipped.
+    delete_all_wavecars: bool
+        if true, all WAVECARs will be deleated after the run
+    address_min_basis: str
+        yaml file including basis set information
+    address_max_basis: str
+        yaml file including basis set information
+    """
+
+    name: str = "lobster"
+    bulk_relax_maker: BaseVaspMaker | None = field(
+        default_factory=lambda: DoubleRelaxMaker.from_relax_maker(RelaxMaker())
+    )
+    vasp_lobster_maker: BaseVaspMaker = field(
+        default_factory=lambda: VaspLobsterMaker()
+    )
+    additional_static_run_maker: BaseVaspMaker | None = field(
+        default_factory=lambda: StaticMaker(
+            input_set_generator=StaticSetGenerator(
+                user_incar_settings={"LWAVE": True},
+                user_kpoints_settings={"grid_density": 6000},
+            ),
+        )
+    )
+    lobstermaker: BaseVaspMaker | None = PureLobsterMaker()
```

Comment:
Just noticed this, you need to use the default_factory and please can you add this to the docstring.
```suggestion
    lobstermaker: BaseVaspMaker | None = field(
        default_factory=lambda: PureLobsterMaker()
    )
```